src/client/components/App/App.js

```diff
@@ -5,6 +5,11 @@ import Header from '../Header';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import HomePage from '../HomePage';
 import AboutPage from '../AboutPage';
+import GalleryPage from '../GalleryPage';
+import TextsPage from '../TextsPage';
+import PressPack from '../PressPack';
```

Comment:
To też strona? Nie powinna się więc nazywać PressPackPage? :)

Reply:
Nie chciałem takiej długiej nazwy robić. Poprawię.

Reply:
Zmienione